draft-ietf-quic-tls.md

```diff
@@ -216,10 +216,12 @@ Note that this omits the EndOfEarlyData message, which is not used in QUIC (see
 
 Data is protected using a number of encryption levels:
 
-- Plaintext
-- Early Data (0-RTT) Keys
-- Handshake Keys
-- Application Data (1-RTT) Keys
+- Initial Keys can be derived by any observer, and so they do not
+  provide cryptographic protection or authentication.
+- Early Data (0-RTT) Keys. These keys are not forward-secure and must protect
+  only idempotent data.
+- Handshake Keys do not authenticate either endpoint.
+- Application Data (1-RTT) Keys provide full authentication and encryption.
 
```

Comment:
I think this looses to overview of what levels exist. This bullet list now replaces an overview with an explanation. Perhaps keep the original list, and add an explanation below.

Reply:
I tend to agree with @mikkelfj here.  The short names are easier to get at without the extra text.

Reply:
OK, this is going away, except for s/Plaintext/Initial